### PR TITLE
add support for force includes in c_cpp_properties.json

### PR DESCRIPTION
**Pull request by bobbrow**
#852

### CODE REVIEW

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/c_cpp_properties.schema.json [hunk 1]
```diff
@@ -47,6 +47,13 @@
                         ],
                         "description": "If set, it overrides the default mode used by the IntelliSense engine. Windows defaults to msvc-x64 and Linux/Mac default to clang-x64."
                     },
+                    "forceIncludes": {
+                        "description": "A list of files that should be included before any include file in a translation unit",
```
Comment (sean-mcmanus) [inline Extension/c_cpp_properties.schema.json]: I think it's supposed to end in a period?

Comment (sean-mcmanus) [reply Extension/c_cpp_properties.schema.json]: I think it should be "forcedIncludes"? That's what VS calls it.

Comment (bobbrow) [reply Extension/c_cpp_properties.schema.json]: VS calls it "forcedInclude".  ok.

**Comment by sean-mcmanus** [comment]
Actually VS uses “Forced Include Files”, so forcedIncludes would be more accurate. Without the “s” it makes it sound like only 1 can be specified.

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
Actually VS uses “Forced Include Files”, so forcedIncludes would be more accurate. Without the “s” it makes it sound like only 1 can be specified.

**Comment by bobbrow** [comment]
VS's CppProperties.json file (which we've modeled c_cpp_properties.json on) spells it `"forcedInclude"`.  I have similar qualms with plurality in the other property names, but if the aim is consistency, it needs to be `"forcedInclude"`

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**